tools/harness-thci/OpenThread.py

```diff
@@ -53,6 +53,9 @@ class OpenThread(IThci):
     LOWEST_POSSIBLE_PARTATION_ID = 0x1
     LINK_QUALITY_CHANGE_TIME = 100
 
+    # only firmware built since thread-reference-20180615(tag) is allowed
```

Comment:
"since" -> "with"?

Also, we don't actually have this tag yet.

Reply:
Yes. Updated the comments. Please have a review. Thanks.